### PR TITLE
Update country data

### PR DESCRIPTION
**Pull request by SebastianSzturo**

### CODE REVIEW

**Comment by SebastianSzturo** [comment]
Thanks again to @seleem1337 for the help ❤️

**Comment by jaimeiniesta** [comment]
Hey, thanks for the update but this was a major change in the API, no more filtering by `:names`, now it's `:unofficial_names` :)

File: lib/data/countries/GB.yaml [hunk 1]
```diff
@@ -10,22 +10,10 @@ address_format: |-
...
-names:
-- United Kingdom
-- The United Kingdom
-- Vereinigtes Königreich
-- Royaume-Uni
-- Reino Unido
-- "イギリス"
-- Verenigd Koninkrijk
-- Great Britain (UK)
+name: United Kingdom of Great Britain and Northern Ireland
```
Comment (jaimeiniesta) [inline lib/data/countries/GB.yaml]: I know this is the official name of the UK, but sadly Google won't geocode it. For example, these 2 will geocode fine:

* 36/38 GREAT CASTLE STREET W1W 8LG LONDON United Kingdom
* 36/38 GREAT CASTLE STREET W1W 8LG LONDON United Kingdom of Great Britain

But this one won't geocode, it won't even return a result on Google maps:

* 36/38 GREAT CASTLE STREET W1W 8LG LONDON United Kingdom of Great Britain and Northern Ireland

For the purposes of this library there's not much we can do, but maybe we can come with an idea, like, for example, adding a new key to the countries data for the "common name"?

**Comment by SebastianSzturo** [comment]
@jaimeiniesta Urgh, this definitely should've been a major version bump 🤦‍♂️

File: lib/data/countries/GB.yaml (hunk 1 above)
Comment (SebastianSzturo) [reply lib/data/countries/GB.yaml]: Hmm I would almost say we should stick to the ISO standard. Which would mean we call it "United Kingdom"?

Comment (jaimeiniesta) [reply lib/data/countries/GB.yaml]: Yes, "United Kingdom" is what I would expect, so I would stick to the ISO standard. But then we should also support "United Kingdom of Great Britain and Northern Ireland", which should then be moved to the `unofficial_names`... while this is in fact the official name 🤔 

So, I don't know, maybe we should explicitly have `iso_name`, and then `official_name`, and `alternative_names` or `other_names`, I don't know.

**Pull request merged**